File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/emsmain/order/OrderRstDTO.cs
using iih.ci.iih.ci.ord.dto.emsmain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.emsmain
{
    public class OrderRstDTO : BaseCiDTO
    {
        
        public FBoolean IsSuccess
        {
            get { return getAttrVal<FBoolean>("IsSuccess", null); }
            set { setAttrVal<FBoolean>("IsSuccess", value); }
        }
        public String MessageInfo
        {
            get { return getAttrVal<String>("MessageInfo", null); }
            set { setAttrVal<String>("MessageInfo", value); }
        }
       
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/d/ems/order/OrderRstDTO.java
package iih.ci.ord.d.ems.order;

import iih.ci.ord.d.ems.base.BaseCiDTO;
import xap.mw.coreitf.d.FBoolean;

public class OrderRstDTO extends BaseCiDTO {
	/**
	 * 医嘱查询返回结果
	 */
	private static final long serialVersionUID = 1L;
	
	/**
	 * 成功标志
	 * @return
	 */
	public FBoolean getIsSuccess() {
		return ((FBoolean) getAttrVal("IsSuccess"));
	}
	/**
	 * 成功标志
	 * @param 真假值
	 */
	public void setIsSuccess(FBoolean IsSuccess) {
		setAttrVal("IsSuccess", IsSuccess);
	}
	/**
	 * 信息
	 * @return
	 */
	public String getMessageInfo() {
		return ((String) getAttrVal("MessageInfo"));
	}
	/**
	 * 信息
	 * @param 信息
	 */
	public void setMessageInfo(String MessageInfo) {
		setAttrVal("MessageInfo", MessageInfo);
	}
	
	
}
